chaos-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: chaos-exporter
+    release: prometheus-stack
```

Comment:
do we want this label present? could be a little confusing wrt helm release, if yes, can we rename, also let's use it in its service also.